### PR TITLE
Added explicit code coverage example to README.md

### PR DESCRIPTION
**Pull request by JoshuaKGoldberg**
Documentation followup to #13.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,6 +67,14 @@ Pass the Chrome **--no-sandbox** and **--disable-setuid-sandbox** arguments:
 $ mocha-headless-chrome -f test.html -a no-sandbox -a disable-setuid-sandbox
 ```
 
+Instrument `src/` code with `istanbul`, run tests on it, and coverage the generated `coverage.json` to an HTML report:
```
Comment (dima117) [inline README.md]: It is not enough information about that test page should use instrumented code.

File: README.md [hunk 2]
```diff
@@ -67,6 +67,14 @@ Pass the Chrome **--no-sandbox** and **--disable-setuid-sandbox** arguments:
...
+
+```
+$ istanbul instrument src -o instrumented
+$ mocha-headless-chrome -f test.html
```
Comment (dima117) [inline README.md]: missing `-c` option

**Comment by JoshuaKGoldberg** [comment]
@dima117 I think I fixed for your feedback?

**Comment by dima117** [comment]
Hi,

I temporarily rolled back your changes from the `master` branch. Causes:
- the example uses an obsolete `istanbul` utility that has not been updated for 2 years. As far as I understand, the actual tool is [nyc](https://github.com/istanbuljs/nyc). I need time to test and adapt your example for the `nyc`.
- Also I'm now looking at the built-in chrome capabilities to generate coverage reports that work without third-party utilities.

Your changes are in the [coverage](https://github.com/direct-adv-interfaces/mocha-headless-chrome/tree/coverage) branch now. Feel free to contact me if necessary.

**Pull request merged**